eg/google-drive-ls

```diff
@@ -15,7 +15,11 @@ Log::Log4perl->easy_init( {
 
 my $gd = Net::Google::Drive::Simple->new();
 
-my $gd_dir = "/top";
+my $gd_dir = @ARGV[0];
```

Comment:
this should use `$ARGV[0]`
why not doing something like `my $gd_dir = $ARGV[0] // q[/top]` which is probably better written in two lines to preserve compatibility with older versions of Perl
```
my $gd_dir = $ARGV[0];
$gd_dir = q[/top] unless defined $gd_dir;
```

Reply:
"top" isn't a synonym for the root directory - the current google-drive-ls is hard-coded to list a folder of that name, and it will fail if it's not found.